beam-lib/src/ids.rs

```diff
@@ -129,6 +129,7 @@ fn strip_broker_id(id: &str) -> Result<&str, BeamIdError> {
     }
 }
 
+#[cfg(feature = "strict-ids")]
```

Comment:
Is the struct not required generally? I was under the impression, that `strict-ids` only handle the consistency checks, esp. BrokerID checks?